tensorflow/core/grappler/optimizers/BUILD

```diff
@@ -869,7 +869,7 @@ tf_kernel_library(
     ],
 )
 
-tf_cuda_cc_test(
+tf_cc_test(
```

Comment:
This will prevent all tests hidden behind #if GOOGLE_CUDA from running, and there are few that are important.

Reply:
@ezhulenev thanks  for the input. I think we have a solution that addresses the issue in tf_cuda_cc_test and doesn't require changing from tf_cuda_cc_test to  tf_cc_test.  Testing it now, I will submit a new PR.